reports/m1_proposal.md

```diff
@@ -0,0 +1,21 @@
+**Description of the data**
+
+The original raw data is downloaded from this URL <https://thepulseofthenation.com/#future>. This dataset is the pull result from Card Against Humanity to the nationally representative of American public about popular social concerns during time period August 1st to 15th 2018. The survey was conducted via face to face, cell phone, and land lines, and respondents' demographic info and opinion choices on survey questions were collected.
+
+The raw dataset contains 800 x 24 datapoints, including collected answers from 800 respondents about 7 demographical questions and 14 opinion questions. It contains 11% of missing data points mainly from redundant 'other' columns.
```

Comment:
Suggested correction: We will analyze data gathered from a sample of 800 participants. Each participant provided responses to approximately 7 demographic questions and 14 opinion-based questions. The dataset had approximately 11% missing values, which were addressed by substituting them with the 'DK/REF' category.

Reply:
We are actually not substituting them with 'DK/REF'; the missing values are from the two 'other' columns and we removed these two columns considering its not containing much of useful info.

Reply:
Ah I see, okay makes sense